lib/index.js

```diff
@@ -304,7 +355,7 @@ export default class HyperList {
     }
 
     if (config.applyPatch) {
-      return config.applyPatch(element, fragment)
+      return config.applyPatch(element, fragment, scroller)
     }
 
     element.innerHTML = ''
```

Comment:
If I understand this correctly, React is not DOM-friendly? I mean that for react you need some kind of virtual dom based on objects? Isn't there a way to transform real DOM elements to react-like dom outside this script?